packages/core/src/App/Components/Elements/NotificationMessage/dp2p-banner.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Icon, StaticUrl } from '@deriv/components';
+import { getUrlBase } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+
+const DP2PBanner = ({ onClick, header, message }) => (
+    <div className='dp2p-banner'>
+        <div className='dp2p-banner__left'>
+            <div className='dp2p-banner__title'>
+                <Localize i18n_default_text={header} />
+            </div>
+            <div className='dp2p-banner__description'>
+                <Localize i18n_default_text={message} />
+            </div>
+            <StaticUrl href='/cashier/p2p/v1'>
+                <Button className='dp2p-banner__btn-accept' secondary onClick={onClick}>
+                    {localize('Learn more')}
+                </Button>
+            </StaticUrl>
+        </div>
+        <div className='dp2p-banner__right'>
```

Comment:
same here, should be `--right`

Reply:
or you can change the naming altogether if it's not a modifier :D right now it sounds like a modifier

Reply:
Alright, got it.